test/controllers/basic_search_controller_test.rb

```diff
@@ -6,6 +6,48 @@ class BasicSearchControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
 
     assert_select 'form#basic-search', { count: 1 }
+
```

Comment:
Non-blocking: we may want to rename this file and class to `search_controller_test.rb` and `SearchControllerTest` to avoid future confusion.